XamarinCommunityToolkitSample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj

```diff
@@ -31,7 +31,6 @@
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <AndroidLinkMode>SdkOnly</AndroidLinkMode>
```

Comment:
Same for this one, did you do this for a reason?

Reply:
This happened automatically

Reply:
okay sure i will revert it back

Reply:
Something that is worth noting HotReload only works when the linker is set to non so with that line HotReload will not work

Reply:
Just to point out that is important. I did that change to make it easy for us to see any linker trouble early. More info [here](https://github.com/xamarin/XamarinCommunityToolkit/pull/437#discussion_r503234521)